static/app/views/releases/list/header.tsx

```diff
@@ -0,0 +1,102 @@
+import {useState} from 'react';
+import {InjectedRouter} from 'react-router';
+import styled from '@emotion/styled';
+
+import * as Layout from 'sentry/components/layouts/thirds';
+import {PageHeadingQuestionTooltip} from 'sentry/components/pageHeadingQuestionTooltip';
+import {TabList, Tabs} from 'sentry/components/tabs';
+import {Tooltip} from 'sentry/components/tooltip';
+import {SLOW_TOOLTIP_DELAY} from 'sentry/constants';
+import {t} from 'sentry/locale';
+import {Organization} from 'sentry/types';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+
+import {MONITOR_VIEW, THRESHOLDS_VIEW} from '../utils/constants';
+
+type Props = {
+  hasV2ReleaseUIEnabled: boolean;
+  organization: Organization;
+  router: InjectedRouter;
+};
+
+function Header({router, hasV2ReleaseUIEnabled}: Props) {
+  const [selected, setSelected] = useState(router.location.query.view);
```

Comment:
this page should be its own route and have its own path in the router instead of using query parameters to control the view

Reply:
hmmmmm that's fair.

complicates this implementation slightly then...

Reply:
actually - i think i'm going to keep this as a query param.

I see the performance page tabs are also utilizing the query param for tab content.
Digging into this - i don't know if i agree that the content here shoudl live within its own route. Just complicates things and makes things messier

Reply:
re: discussion in slack
going to go with the separate root level route instead